packages/riverpod/lib/src/framework/provider_base.dart

```diff
@@ -213,6 +215,21 @@ abstract class ProviderElementBase<State> implements Ref<State>, Node {
 
   var _debugSkipNotifyListenersAsserts = false;
 
+  String? _debugId;
+
+  /// A unique ID for this object, used by the devtool to differentiate two [ProviderElementBase].
+  ///
+  /// Should not be used.
+  String get debugId {
```

Comment:
The ID should be initialized inside `mount`, not on first read

Reply:
Cool! Will do that.

Reply:
Changed!